.github/workflows/e2e.yaml

```diff
@@ -30,11 +30,34 @@ jobs:
           path: kmm_local.tar
           retention-days: 1
 
+  build-signing-image:
+    runs-on: ubuntu-latest
+
+    name: Build the signing image
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Build the image
+        run:  make signimage-build SIGNER_IMG=kmm-signimage:local
+
+      - name: Export the image
+        run: docker save -o kmm-signimage_local.tar kmm-signimage:local
```

Comment:
How about
```suggestion
        run: docker save -o kmm_signimage_local.tar kmm-signimage:local
```
?

Reply:
Let's keep the dash like in the original image name.